### ISSUE TITLE
What is the problem with this tacotron training in other language?

### ISSUE THREAD

**Issue by Adibian** [NVIDIA/tacotron2#525]
I want to train Tacotron in Persian language and I follow [this training](https://github.com/NVIDIA/tacotron2#training-using-a-pre-trained-model). I have about 2500 wav files (about 2 hours) and their text and phonemes. Because of phonemes are in English characters I use them as texts of the wav files in training. Then start training Tacotron for 20 epochs (about 4 hours) and it reached loss 0.6 . After that I use[ this inference](https://github.com/NVIDIA/tacotron2/blob/master/inference.ipynb) file to get results of tests. But I get this results:
![results](https://user-images.githubusercontent.com/42696179/135856145-0ec2d395-d72f-490e-88f1-7c3155b5ee80.png)
and the resulted audio is just some noise..
I tried to use original text file and results don't change. 
what's the problem with that? what should I do?

**Comment by fancat-programer** [comment]
Too little data for training

**Comment by Adibian** [comment]
I finally understand what was the problem. I just must train the model more..
After about 200 epochs (10 times more) results became so good (for this small data).
![output(28000)(1633558896 3588855)](https://user-images.githubusercontent.com/42696179/136669743-36e0381f-d2d5-41e2-ad43-95e398b2a8de.jpg)
